resources/views/projects.blade.php

```diff
@@ -6,11 +6,35 @@
 @section('content')
 <div class="padded content">
 	<h1>Projects</h1>
+	<!-- Filters -->
+	<form class="ui form">
+		<!-- Search by name -->
+		<div class="field">
+			<label for="searchName">Name</label>
+			<input id="searchName" type="text" name="searchName" placeholder="e.g. A cool project">
+		</div>
+		<!-- Search by tag -->
+		<div class="field">
+			<label for="tags">Tags</label>
+			<select id="searchTags" name="tags" class="ui fluid search dropdown searchTags" multiple>
+				@foreach ($tags as $tag)
+					<option value={{ $tag->id }}>{{ $tag->name }}</option>
+				@endforeach
+			</select>
+		</div>
+		<button id="searchButton" type="button" class="ui primary submit button" onclick="filterProjects()">Search</button>
```

Comment:
Let's remove `onclick="filterProjects()"` since we're no longer adding this on this PR.